### PR TITLE
Remove remaining legacy serializer

### PR DESCRIPTION
**Pull request by pbiggar**
Changelog:

```
StdLib
- Remove toString
- Add Bool.toString
- Add Int.toString
- Add Char.toString
- Add Uuid.toString
- Add Float.toString

Internal:
- Remove DvalReprLegacyExternal.toEnduserReadable serializer
```

### CODE REVIEW

File: backend/src/LibExecutionStdLib/LibChar.fs [hunk 1]
```diff
@@ -143,4 +143,20 @@ let fns : List<BuiltInFn> =
...
+      fn =
+        (function
+        | _, [ DChar c ] -> Ply(DStr(c))
+        | _ -> incorrectArgs ())
+      sqlSpec = NotQueryable
+      previewable = Pure
+      deprecated = NotDeprecated }
+
+
+    ]
```
Comment (StachuDotNet) [inline backend/src/LibExecutionStdLib/LibChar.fs:162]: Is this intended to be a new pattern of ending these fn lists with 2 newlines? (asking as a few files have this)

File: backend/src/LibExecutionStdLib/LibBool.fs [hunk 2]
```diff
@@ -64,4 +64,22 @@ let fns : List<BuiltInFn> =
...
+      returnType = TStr
+      description = "Return {\"true\"} or {\"false\"}"
+      fn =
+        (function
+        | _, [ DBool b ] ->
+          match b with
+          | true -> Ply(DStr("true"))
+          | false -> Ply(DStr("false"))
+        | _ -> incorrectArgs ())
+      sqlSpec = NotQueryable
```
Comment (StachuDotNet) [inline backend/src/LibExecutionStdLib/LibBool.fs:81]: Probably doable

Comment (StachuDotNet) [reply backend/src/LibExecutionStdLib/LibBool.fs:81]: adding a tiny PR to mark these as NotYetImplemented

Comment (StachuDotNet) [reply backend/src/LibExecutionStdLib/LibBool.fs:81]: actually, what do you think? Am I right to think they should be marked as NotYetImplemented?

Comment (pbiggar) [reply backend/src/LibExecutionStdLib/LibBool.fs:81]: Yeah, NotYetImplemented is fine.

File: backend/src/LibExecutionStdLib/LibChar.fs (hunk 1 above)
Comment (pbiggar) [reply backend/src/LibExecutionStdLib/LibChar.fs:162]: I'm sorta undecided here. It's annoying having the list end at the last entry, but ugly to add space. If you have a preference, happy to go with it.

Comment (StachuDotNet) [reply backend/src/LibExecutionStdLib/LibChar.fs:162]: I've been fine with the `} ]` endings. I've played with both locally, and find the empty lines between the two sort of ugly, not worth the tiny diff annoyance when adding a stdlib fn 'at the end'

**Pull request merged**